### DISCUSSION TITLE
How to save and retrieve a JSON directly in latest version of RealmDB-Kotlin in Android during runtime without RealmObject Using RealmDB - Kotlin

### DISCUSSION THREAD

**Discussion by RageshAntonyHM** [realm/realm-kotlin#1790] [Q&A]
I need to store and retrieve a JSON in ReamDB in runtime without creating a RealmObject for that. Also want to check whether the particular JSON object is present in the DB already using a giving field (like “id”). If it exists, then update the object,If not, then save it as new object.

How to do this in latest version of RealmDB  Kotlin SDK in Android.

I searched internet but all examples throw “unresolved references” for classes. It seems they are deprecated and removed. Also, most internet articles are using Java SDK or Realm-Android(a java extn for Android) but I need Kotlin SDK which also targets Kotlin Multi platform. 

Please help me.

**Comment by BlueCobold** [comment]
Since opening Realms in RealmKotlin requires to hand over the set of classes involved in this session, I don't think you can get away without any RealmObject classes like in Realm for .Net or RealmJava for example.
